client/components/deposits-overview/strings.ts

```diff
@@ -5,4 +5,11 @@ import { __ } from '@wordpress/i18n';
 
 export default {
 	heading: __( 'Deposits', 'woocommerce-payments' ),
+	next_deposits: {
```

Comment:
Nit/opinion: 😉 I suggest using camelCase in JS/TS where possible, for consistency.  This is enforced by [`.eslintrc`](https://github.com/Automattic/woocommerce-payments/blob/e572f566a846088b23a0b3b6926996e22f679ab4/.eslintrc#L37-L43) for variable naming, but allowed for object properties.
```suggestion
	nextDeposits: {
```